extensions/ql-vscode/src/legacy-query-server/run-queries.ts

```diff
@@ -308,12 +312,19 @@ export async function compileAndRunQueryAgainstDatabaseCore(
   query: CoreQueryTarget,
   generateEvalLog: boolean,
   additionalPacks: string[],
+  extensionPacks: string[] | undefined,
   outputDir: QueryOutputDir,
   progress: ProgressCallback,
   token: CancellationToken,
   templates: Record<string, string> | undefined,
   logger: Logger,
 ): Promise<CoreQueryResults> {
+  if (extensionPacks !== undefined && extensionPacks.length > 0) {
+    await showAndLogWarningMessage(
+      "Legacy query server does not support extension packs.",
```

Comment:
Possibly add a message here to suggest changing the `extensionPacks` configuration property to `none`.

Though, so few people are using the legacy query server now that this is probably not a big deal.

Reply:
I think we can probably leave it as-is. Once I add the debug configuration in my next PR, I'd just have to update the warning again with more detail about changing the `extensionPacks:` property in `launch.json`, so let's just hope that nobody who is still using the legacy query server is also on the bleeding edge of extension pack development:)